### ISSUE TITLE
Remove RocketPool as the recommended staking solution

### ISSUE THREAD

**Issue by tsudmi** [ethereum/ethereum-org-website#5633]
Placing RocketPool as recommended staking solution for users with less than 32 ETH breaks the idea that Ethereum Foundation should stay neutral to the protocols that build on top. There should be a link to all the solutions out there, and the user should be able to evaluate all of them and choose the one he likes the most.

**Comment by tsudmi** [comment]
Reverts https://github.com/ethereum/ethereum-org-website/pull/5349

**Comment by djrtwo** [comment]
Agreed that rocketpool should not be *independently* highlighted like this on the ethereum.org website.
I think the page should either give a better range of possibilities or not address the issue at all

**Comment by corwintines** [comment]
At the risk of looking dumb here, what other staking services let you run a validator with less than 32 ETH? Going to do some looking on my own, but any direction on this would be appreciated. I do think we should flush this section out more, but I also think Rocket Pool ended up getting some consideration due to you being able to run a validator with less than 32 ETH as well.

**Comment by samajammin** [comment]
Hey @tsudmi - thanks for creating this issue & thank you folks for weighing in!

**First, covering your point neutrality:**

> Placing RocketPool as recommended staking solution for users with less than 32 ETH breaks the idea that Ethereum Foundation should stay neutral to the protocols that build on top. 

While Ethereum as a protocol is neutral, [the ethereum.org team](https://ethereum.org/en/about/) (which doesn’t represent any official stance of the Ethereum Foundation) is admittedly opinionated when it comes to ethereum.org content. I believe we have to be opinionated to a certain degree. For instance, we have listing criteria for how we curate different products & projects who want to be added to the website:
- [Adding products](https://ethereum.org/en/contributing/adding-products/)
- [Adding developer tools](https://ethereum.org/en/contributing/adding-developer-tools/)
- [Adding exchanges](https://ethereum.org/en/contributing/adding-exchanges/)
- [Adding content resources](https://ethereum.org/en/contributing/content-resources/)

Whether it's listing Ethereum wallets, exchanges, clients, staking providers, or developer tools, this is a type of decision we have to consider often. Ultimately it comes down to a balance of staying completely neutral vs. taking an opinionated stance when we believe it delivers a better experience for our users. Our basic goal is to accept the fact many decisions are ultimately subjective and to provide transparency around those decisions so we can collect feedback & iterate.

All that said, we do strive to be credibly neutral with [ethereum.org](http://ethereum.org) in the sense that we are not monetarily incentivized to highlight any particular project or product. We don’t accept any affiliate fees, or attempt to make revenue in any way. The only motivation we have is to create the most useful content & the best UX as possible for people onboarding into the Ethereum ecosystem. Inevitably that requires subjective opinions.

While we try to remain unbiased where possible, over the past couple of years the overwhelming feedback we continue to receive from the community & user testing is that we should be more opinionated into order to improve UX. Decision fatigue is real. The paradox of choice is real. Users have told us they don’t want a [list of 40 wallets to choose from](https://ethereum.org/en/wallets/find-wallet/), or a [list of 37 unvetted staking providers](https://beaconcha.in/stakingServices). We plan to continue to iterate on how to best balance fair treatment of legitimate projects & the desire from users to receive a basic level of curation & recommendations.

> There should be a link to all the solutions out there, and the user should be able to evaluate all of them and choose the one he likes the most.

We still provide a link to beaconcha.in with all the solutions out there. Users can still evaluate all of them. While I didn’t directly approve the community PR (https://github.com/ethereum/ethereum-org-website/pull/5349), I agree with our team’s decision to merge it. RocketPool seems to be the unanimous top recommendation from most experts (like the EthStaker core team), so we think the update improved the page. It didn’t take remove any content such as the link to beaconchain’s list. We also knew we'd be updating the page soon anyway, which brings us to...

**Second, providing context on our current staking content plans. Input welcome!**

Our staking resources are something we’re actively working to improve right now (as you can see on our Q1 roadmap: #5105). Our primary goals are to expand our resources around both pooled staking & solo staking to make this process easier to understand & easier to accomplish for our site visitors.

Our new designers (hi @konopkja & @nloureiro!) just started last week, so it’s early stages, but they're currently working on a redesign of this page. Happy to share designs & content on this soon to gather feedback, so stay tuned.

We’re also looking into:
1. Creating a listing criteria for staking providers.
2. Building out more documentation on how the process of staking works as well a risks & considerations when doing it. This may evolve into a “docs” section similar to [our current developer docs](https://ethereum.org/en/developers/docs/). 

We’d love to collaborate with any staking experts out there, so feel free to comment here or [drop in our Discord](https://discord.gg/CetY6Y4) if you’re interested in adding input or getting involved! I'll leave this issue open for discussion until we ship that updated page.

**Comment by samajammin** [comment]
> what other staking services let you run a validator with less than 32 ETH?

@corwintines the list we currently link to on our staking page provides info on the minimum stake required for many services:
https://beaconcha.in/stakingServices

**Comment by hanniabu** [comment]
@timbeiko The page clearly lists a link to other staking providers and is in no way giving the impression that Rocket Pool is the only solution. This issue was brought up by a tweet (by tsudmi) that intentionally cropped out the link to mislead the crypto community and drive an angry mob your way.

Given the marketshare of Lido and other centralized services along with their marketing and purchasing power, listing Rocket Pool (the only truly decentralized staking pool solution) is the responsible and prudent thing at the time. Rocket Pool has notoriously been helping support client diversity and is responsible for >10% (13.8%?) of consensus nodes despite having only 1.1% of staked ETH.

**Comment by flaskr** [comment]
> At the risk of looking dumb here, what other staking services let you run a validator with less than 32 ETH? Going to do some looking on my own, but any direction on this would be appreciated. I do think we should flush this section out more, but I also think Rocket Pool ended up getting some consideration due to you being able to run a validator with less than 32 ETH as well.

That's a good point - a lot of services allow you to 'stake', but not run a validator (permissionlessly or not) on their behalf.

**Comment by kiriyha** [comment]
@samajammin It seems like even the opinions of credibly neutral ETH2 heavyweights who commented on this issue cannot sway your opinion. Your team also seems to have ignored your own guidelines regarding the minimum period that the project should be operational before it gets highlighted. Finally, if the minipool concept (ie run your own node with <32 ETH) is so great, why Ankr (who were running with this concept for almost a year before RP) was never mentioned anywhere? Seems like a lot of DD was performed before making changes on the page. Nice. 

If the Ethereum Foundation is OK with you taking this biased approach then I guess there is nothing to discuss. Personally, watching this unfold has been incredibly frustrating. I hope you realize that this is how you get devoted Ethereum builders to become disillusioned with our collective mission.

**Comment by flaskr** [comment]
> Finally, if the minipool concept (ie run your own node with <32 ETH) is so great, why Ankr (who were running with this concept for almost a year before RP) was never mentioned anywhere? Seems like a lot of DD was performed before making changes on the page. Nice.

Have you done your own DD on Ankr? Last I checked a few months ago, they were very loud on how you could run your own validators, but the docs were missing a lot of information and there was no way someone could just come in and spin up something themselves.

**Comment by eccentricexit** [comment]
Ethereum foundation not only should keep rocketpool there but also add a note saying something like:

“We discourage using centralized staking services as those degrade the decentralization and censorship resistance of the network. Instead, we recommend using a permissionless, decentralized protocol like rocketpool since it increases it.”

If the website was choosing a specific company, then it would be biased.

This is not the case here. Don’t be fooled into thinking rocketpool is the same as the others.

**Comment by AD1528** [comment]
Context is important.  

Before mentioning any particular service or protocol (or, ideally, a curated group of offerings, rather than just one), It would actually be more helpful to state some of the values that benefit the health and resilience of Ethereum overall.  

E.g. a section intro that talks about client diversity and increasing the number of validators could be added to better educate those landing on this page and help inform their decision?

**Comment by samajammin** [comment]
Appreciate the additional input folks! 

FYI I created a couple more issues aiming to address much of your feedback & to improve our listing of staking services. Encourage you to give those a read & weigh in if you have input:
- Revamp the Staking page #5647
- Create a listing policy for staking services #5646

**Comment by tsudmi** [comment]
@samajammin I still think we should temporarily remove rocketpool from that page and bring it back once the staking page will be revamped. Currently, there are multiple wallets user can choose from, same goes for dapps and only one staking solution if you have less than 32 ETH.

**Comment by yokem55** [comment]
I'm going to advocate a somewhat different position: ethereum.org should not have a 'staking services' page at all. 

Just 'staking' in and of itself is at best a neutral thing for Ethereum. Just throwing ether over the wall into the deposit contract looking for a return does not by itself do anything good for the protocol. What does help the protocol is getting more distinct and independent individuals *operating validating nodes.* 

So, instead of a 'staking services' page. There should instead be a 'validating node tools' to help more people **operate validating nodes on their own**.

Ie, tools like the Wagyu toolset, Dappnode and Stereum for better user interfaces for running a node. Hardware vendors like Avado who can sell preconfigured/built hardware bundles for those who need help on the hardware side. Node hosting services like AllNodes for folks who don't have reliable internet. A list of up-to-date and well written and edited guides for Windows and Mac users.  And yes, RocketPool as a tool to enable folks with less then 32 ether to run a validating node. 

If folks really just want a 'staking services' page, a link to the Beaconcha.in list is more then sufficient.

**Comment by gakonst** [comment]
+1 on reverting. candidly [this](https://ethereum.org/en/staking/) reads like an ad which doesn't sound fitting for ethereum dot org 

> Stake in a pool
> You can stake on Rocket Pool by swapping ETH for Rocket Pool's liquid token rETH with as little as 0.01 ETH. Want to run a node? You can also run a staking node on Rocket Pool starting from 16 ETH. The remaining ETH gets assigned from the Rocket Pool protocol, using ETH users have deposited to the pool.

**Comment by jasperghost1** [comment]
> the discussion around decentralization seems subjective/poorly specified

You're right, I think Sam was doing other services a favor when he wrote;

> While I didn’t directly approve the community PR (https://github.com/ethereum/ethereum-org-website/pull/5349), I agree with our team’s decision to merge it. **RocketPool seems to be the unanimous top recommendation from most experts (like the EthStaker core team**), so we think the update improved the page

The discussion was vague because it's been readily fleshed out in other forums. There are three simple axes, principally, we ought to define our staking services by as these are the same standards we hold Ethereum itself to. Rocket Pool is the clear leader in all of them. 

Permissionlessness - anyone can spin up an Ethereum node without KYC or any loopholes. 

Decentralized - the whole network shouldn't have central points of failure

Trustless - no counterparty risk 

We can start at the top, there aren't any other permissionless protocols. You can apply through a governance process over quite some time and put up (likely) large capital bonds, but that's a far, far cry from permisionless. 

Decentralized? No service has provided more individual node operators than Rocket Pool. Combine all the rest of them and they'd be less than what Rocket Pool has put together in a few months. Not just that, but in the most diverse client set of any (https://pools.invis.cloud/). Over 800 nodes, at least 400 unique operators, across every time zone on Earth. 

Rocket Pool isn't perfect with trust. There are upgradable contracts that node operators can opt into, there's an oracle DAO that controls certain parameters, however, it is trust minimized compared to the field. 

We can talk about expanding the services out, but why would you want to harm the only product that actually is healthy for the network? It's clearly in Ethereum's best interests and that's what Ethereum.org ought care about.

**Comment by danielgmason** [comment]
> 

Seems helpful to have an option up until the page gets revamped even if it's suboptimal, but agree that something that presents more information about staking + the myriad of providers with pros/cons is definitely better.

**Comment by samajammin** [comment]
Again, appreciate the input on this folks 👌 

Overall I think much of the pushback on https://github.com/ethereum/ethereum-org-website/pull/5349 is valid but rather than focus on "remove RocketPool" I think it's more productive to focus on how to display multiple staking options. 

@nloureiro just shared some design prototypes of how we might accomplish that. I encourage you all to check it out & add your feedback: https://github.com/ethereum/ethereum-org-website/issues/5647#issuecomment-1068394781

I'm going to close this issue out but hope to keep the conversation going. I encourage you all to bring further discussion on this topic to #5647

**Comment by gakonst** [comment]
@samajammin thanks for the follow up! The direction towards revamping the page seems good! Given that, do you not think it makes sense to move forward with https://github.com/ethereum/ethereum-org-website/pull/5635, since #5647 ACKs that the previous design was not clear enough, and that the original https://github.com/ethereum/ethereum-org-website/pull/5349 PR was not the "right" solution?
